org.eevolution.hr_and_payroll/src/main/java/base/org/eevolution/model/MHRProcess.java

```diff
@@ -977,6 +977,8 @@ private void createMovements() throws Exception
 		scriptCtx.put("_From", dateFrom);
 		scriptCtx.put("_To", dateTo);
 		scriptCtx.put("_Period", payrollPeriod.getPeriodNo());
+		scriptCtx.put("_PeriodNo", payrollPeriod.getPeriodNo());
```

Comment:
Hi @SusanneCalderon I suggest change it according with #761
The **_Period** is representative of **PeriodNo** and for old compatibility is better keep, but you can add the follow keys I think:

Example:
**_HR_Period_ID**: Period Id

The problem was that in beginning the keys was created without a good standard definition but I think that we should keep old implementation wihtout problems

Reply:
Hi @yamelsenih 
Theese are the keys now for period values:

		scriptCtx.put("_Period", payrollPeriod.getPeriodNo());
		scriptCtx.put("_PeriodNo", payrollPeriod.getPeriodNo());
		scriptCtx.put("_PeriodID", getHR_Period_ID());